blog/2022-08-19-jit-service account/index.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: jit-service-account
+title: "Understanding Just-in-time (JIT) access with service accounts in Paralus"
+authors: [atul]
+tags: [paralus]
+---
+
+Organizations today are focusing a lot on scalability and flexibility of their applications. Most of the age-old monolith applications are now being converted to more modern and agile microservices which reside in containers.
+
+As more and more applications are containerized, orchestration tools like Kubernetes are at the helm. However, with every new resource addition, there's an overhead. And the one that we are going to focus on today is **access management for Kubernetes**.
+
+<!--truncate-->
+
+With hundreds of containers on different clusters spread across multiple on-prem and cloud platforms, managing access for every user to every resource is tough. Especially when many resources like pods are ephemeral, access management is a challenge. Further, the team dynamics keep changing and you have new members coming in and moving out. If access is not revoked for an outgoing user, it poses a security threat.
+
+While Kubernetes does provide support for namespaces and role based access control (RBAC), it's not enough especially for large, enterprise grade production environments. That's when the need for a Just-In-Time access solution was required.
+
+## What is Just-in-time access?
+
+For the unaware, the just-in-time philosophy was created by folks at [Toyota](https://global.toyota/en/company/vision-and-philosophy/production-system/) & was implemented in their supply chains. Today, this philosophy is used across industries and verticals to make the processes more streamlined and agile. Just-in-time access too follows the same philosophy but for managing access.
+
+Just in time access refers to provisioning of access for a user to a particular resource which is valid only for a specific duration. A typical JIT access workflow starts with the user requesting access for a resource, the system validates that against an access policy and in turn provides access to the requested resource for a specific duration.
+
+For a team with multiple users and resources, implementing JIT access is a critical security control.
+
+## How Just In Time access works in Paralus
+
+Paralus allows you to import and manage Kubernetes clusters hosted on different platforms from a single dashboard. It provides you to configure Zero Trust access for your users to your Kubernetes resources. Paralus also comes with Just in time service accounts creation on the target cluster which is valid only for a certain duration. To understand how just in time access works in Paralus with Service accounts, refer to the image below.
```

Comment:
what image are we referring to here ?

Reply:
I need to create an image based on the steps mentioned in the next section - basically show all the steps for JIT in a diagram. Will share once I have something.

Reply:
Added an image.